src/plugins/console/server/lib/spec_definitions/json/generated/search.json

```diff
@@ -69,6 +69,6 @@
       "{indices}/_search",
       "{indices}/{type}/_search"
     ],
-    "documentation": "https://www.opensearch.org/guide/en/elasticsearch/reference/master/search-search.html"
+    "documentation": "https://opensearch.org/docs/latest/opensearch/search-template/"
```

Comment:
I don't think the search-template page is the right documentation. It doesn't really seem like the Search API is fully documented. I think we should add this to `noDocumentation` but maybe link to https://opensearch.org/docs/latest/opensearch/ux/?